packages/easy-coding-standard/README.md

```diff
@@ -31,11 +31,15 @@ Are you already using another tool?
 
 ## Install
 
+### Regular Version
```

Comment:
This headline don't add any extra information. It's standard to have composer install by default, no need to write it.